### DISCUSSION TITLE
custom font size

### DISCUSSION THREAD

**Discussion by VITTHANI** [otfried/ipe#535] [Ideas]
as there are inbuilt commands for font size in TeX, it is worth extending by allowing desire point size or with any other valid measure. I do that using arguing explicit command such as:
`\fontsize{size}{baselineskip}\selectfont`,
instead of writing this, it can be added as a GUI user input, for xelatex and lualatex, as pdflatex does not support this.

**Comment by otfried** [comment]
Do you know that you can set an arbitrary absolute font size by pressing on the "A" button to the left of the text size selector?

**Reply by VITTHANI** [reply]
I should have ask the question first, rather than making a statement. I apologize. you should delete this discussion, i guess.
